tasks/leave_restart_consul.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+- name: Consul leave
+  delegate_to: "{{ item }}"
+  command: "{{ consul_binary }} leave {% if consul_acl_enable %} -token {{ consul_acl_master_token }} {% endif %} -http-addr {{ consul_addresses_http }}:{{ consul_ports_http }}"
+  changed_when: false
```

Comment:
Since this changes system state, I would argue it should be `changed_when: true`.

Reply:
I agree :+1: I just removed the `changed_when`, as it should be the same as set it to true.